
---
float64[6] pos 